### ISSUE TITLE
Unable to access lineage pages for genus Homo

### ISSUE THREAD

**Issue by ensatina** [keesey/phylopic#23]
I am unable to access the lineage pages for the genus Homo and species Homo sapiens. For example,

[https://www.phylopic.org/nodes/1ee65cf3-53db-4a52-9960-a9f7093d845d/homo-sapiens-silhouettes](https://www.phylopic.org/nodes/1ee65cf3-53db-4a52-9960-a9f7093d845d/homo-sapiens-silhouettes)

brings up an Incertae Sedis error.

**Comment by keesey** [comment]
Confirmed. Trying a rebuild for a temporary fix.

I think possibly it's caching a 5xx error response from the API. Need to look into safeguarding against that.

**Comment by keesey** [comment]
Redeploy did not fix it. Cause is not immediately apparent.

The lineage page actually does work: https://www.phylopic.org/nodes/1ee65cf3-53db-4a52-9960-a9f7093d845d/lineage

**Comment by keesey** [comment]
Page works fine going through tot the origin server, so it looks like something bad got caught in Vercel's cache. https://phylopic-www-git-phylopic-wwwprod-keesey-creations.vercel.app/

**Comment by keesey** [comment]
I deployed a new patch release of the website (`v2.6.1`) and the page is back. I am leaving this ticket open, though, because I do not understand the root cause and I want to see if it happens again.

**Comment by keesey** [comment]
I deployed a possible fix for seeing 404 pages when it should be a server error page. Still monitoring.

**Comment by keesey** [comment]
Nope, didn't fix it. Here's one now: https://www.phylopic.org/nodes/efeb0b7b-30d1-4d20-b3b5-63779a767126/macrolepidoptera-silhouettes

**Comment by keesey** [comment]
I rolled out some other measures that may fix it. Still keeping an eye out.

**Comment by keesey** [comment]
Quick note: still seeing this very infrequently.

**Comment by keesey** [comment]
Quick note: still seeing it infrequently, especially after new builds. Might have to do with the website getting out of sync with the latest build.

**Comment by keesey** [comment]
It is possible that this PR may fix the issue: https://github.com/keesey/phylopic/pull/35

**Comment by keesey** [comment]
I haven't seen this in a long while, so I think we can assume #35 fixed it.
